dashboard/run_dashboard_app.py

```diff
@@ -407,7 +407,7 @@ def get_total_user_chours(
         name, total_chours = app.db.get_total_user_chours(
             hpc, username, start_date, end_date
         )
-        data.append({"username": name, "total_core_hours": total_chours})
+        data.append({"username": USERS[name], "total_core_hours": total_chours})
```

Comment:
Oh, I meant the command run on HPC to display the full name. Is that possible.

There should be options to sreport which does that.

Reply:
reformatted sreport